applications/dashboard/design/admin.css

```diff
@@ -6706,6 +6706,12 @@ label,
   border: 0.0625rem solid #ccc;
 }
 
+.form-sectionTitle {
```

Comment:
I don't think this is correct. There's "Subheading" and "Subheading Block" available in the styleguide:

`/vanilla/applications/dashboard/styleguide/section-1.html`

I think it would be proper to have the "Global Site Colors" as a block and the "Header Variables" as a block.

Reply:
Because this is a `ConfigurationModule` I can't make the sections themselves blocks, but I can remove my custom CSS and re-use `subheading`